.github/workflows/pr-reporting.yml

```diff
@@ -0,0 +1,35 @@
+name: PR Quality Reporting
+
+on:
+  workflow_run:
+    workflows: [Build pr]
+    types:
+      - completed
+    branches: [main, release/**, next/**, feat/add-jest-tests]
```

Comment:
`feat/add-jest-tests` is removed before merge?

Reply:
good call out, thank you!